week_4/route-table-default.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_default_route_table" "week4-default-route-table" {
```

Comment:
It's not a mistake but it's better not to use the default VPC route table, but to create a separate one, as it was done for the public one and make the appropriate route table - subnet association.

Reply:
Got it